.clippy.toml

```diff
@@ -0,0 +1 @@
+msrv = "1.88.0"
```

Comment:
was there a reason for the v1.88 MSRV btw? I used v1.90 in the `rust-toolchain.toml` file and now I'm wondering whether I should've used v1.88 instead 😅